apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.RetryConformanceTests/RetryConformanceTest.cs

```diff
@@ -53,20 +53,16 @@ public async Task RetryTest(RetryTest test)
     {
         Skip.IfNot(ShouldRunTest(test));
 
-        // TODO: Work out what to do about groups. The old CreateMethodMapping 
-        //CreateMethodMapping(test);
-
         foreach (InstructionList instructionList in test.Cases)
         {
             /*
              * Will remove this section in final PR
              * This is to allow testing on cloud run without reseting connection
-             * 
+             */
```

Comment:
Is the plan still to remove this before merging into main, but to merge it into the retry test branch as-is?